packages/kit/src/cli.js

```diff
@@ -153,8 +153,7 @@ prog
 			process.env.NODE_ENV = process.env.NODE_ENV || 'production';
 			process.env.VERBOSE = verbose;
 
-			const svelte_config = await load_config();
-			const vite_config = await get_vite_config(svelte_config);
+			const vite_config = await get_vite_config();
 			await vite.build(vite_config); // TODO when we get rid of config.kit.vite, this can just be vite.build()
```

Comment:
see the TODO — can we get rid of `get_vite_config` (or rather, replace it with a `check_vite_config_exists` or something) and let Vite worry about its own config loading?

Reply:
(would check but i'm in the back of a car, tethering to a flaky hotspot — not very conducive to work)